MAINTAINERS.md

```diff
@@ -9,6 +9,12 @@ See the information about [community membership roles](https://wiki.lfedge.org/d
 | Joe Pearson  | joewxboy    | <joe.pearson@us.ibm.com> |
 | John Walicki | johnwalicki | <johnwalicki@gmail.com>  |
 | Lily Zhang   | liilyZhang  | <zhangl@us.ibm.com>      |
-| Ling Gao     | linggao     | <linggao@us.ibm.com>     |
 | Max McAdam   | maxmcadam   | <maxwell.mcAdam@ibm.com> |
```

Comment:
Can you please move Max McAdam to emeritus too ? He is no longer involved

Reply:
I've submitted another commit which moves Max to the Emeritus list.